packages/components/form/src/field_error.tsx

```diff
@@ -0,0 +1,80 @@
+import { FieldError as HookFormFieldError } from "react-hook-form";
+import snakeCase from "lodash/snakeCase";
+import { BoxProps } from "@buoysoftware/anchor-layout";
+import { TFunction } from "i18next";
+import { useTranslation } from "react-i18next";
+
+import { useFormScope } from "./form_scope_provider";
+import { ErrorMessage } from "./error_message";
+
+export type InputType =
+  | "checkbox"
+  | "collectionCheckbox"
+  | "hidden"
+  | "radio"
+  | "select"
+  | "text";
+
+interface OwnProps {
+  error?: HookFormFieldError;
+  name?: string;
+  inputId: string;
+  inputType?: InputType;
+}
+
+export type FieldErrorProps = OwnProps & BoxProps;
+
+function lookupErrorMessage(
+  error: HookFormFieldError,
+  scope: string,
+  name: string,
+  inputType: InputType,
+  t: TFunction
+): string {
+  if (error.message !== undefined && error.message !== "") {
+    return error.message;
+  }
+
+  const i18nName = snakeCase(name);
+  const i18nErrorScope = snakeCase(scope);
+  const inputMessage = t(`${i18nErrorScope}.${i18nName}.${error.type}`, {
+    defaultValue: "",
+  });
+
+  if (inputMessage !== "") {
+    return inputMessage;
+  }
+
+  const typeMessage = t(`input_types.${snakeCase(inputType)}.${error.type}`, {
+    defaultValue: "",
+  });
+
+  if (typeMessage !== "") {
+    return typeMessage;
+  }
+
+  return t(error.type);
+}
+
+export const FieldError: React.FC<FieldErrorProps> = ({
+  error,
+  inputId,
+  name,
+  inputType = "text",
+  ...elementProps
+}): React.ReactElement | null => {
+  const { t } = useTranslation("forms", { keyPrefix: "errors" });
```

Comment:
Does this need to be updated to leverage the tNamespace form the form level?

Reply:
Good question. I'm not sure. This was pulled verbatim from BuoyRails so it must not be updated in there either.

Reply:
I dug into this a little further to see if I could find anything. As far as I can tell the only place that was updated to utilize `tNamespace` is the `useFormField` hook and this was done in a handful of forms. It's unclear why we ended up going this route as opposed to keeping all the form translations in the single forms namespace.

I'm inclined to actually back out the `tNamespace` feature from `FormScopeProvider` until we are more intentional about the change we might be trying to do here.

Reply:
I take that back, I see now that the way tNamespace is being used is it supports the namespace `forms` as a fallback. I'll go ahead and dig into adding this to FieldError

Reply:
Updated!